src/extension.ts

```diff
@@ -188,6 +205,10 @@ class RunLineInEditor {
     }
 
     private findResultDocument() {
+        if (this.settings.showResponseInDifferentTab) {
```

Comment:
To keep the change simple, just read the setting with workspace.getConfiguration() on line 212 below: `if (this.resultDocument && !workspace.getConfiguration(...)) {`

Reply:
@chrmarti - I'm going to abandon this PR and split the code 3 smaller ones (tabs, doc language, status bar item) based on your comments.

Reply:
Functionality moved to https://github.com/microsoft/vscode-azurecli/pull/55